holidayextras/react-time-select
# PROD-1247 react timeSelect upgrade to react 14

#### What does this PR do? (please provide any background)

This PR upgrades react to react 14 and fixes the component to work with 14.
#### What tests does this PR have?

Unit tests run 'npm test' and make sure they all pass.
#### How can this be tested?

Run the example do npm start and make sure the page has no errors and times can be selected from the list.
#### Screenshots / Screencast

http://grab.by/NCEE
#### What gif best describes how you feel about this work?

![](http://i.imgur.com/qtvgLc4.gif)

---
#### Reviewers

**Review 1**

**Review 2** *

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
